configs/problem/ic_conda18.yaml

```diff
@@ -0,0 +1,53 @@
+# @package _group_
+protocol: 'condda'
+workdir: ''
+harness: 'local'
+domain: image_classification
+test_ids:
+- OND.54011215.0000.1236
+hints:
+- red_light
+dataset_root: ""
+detectors:
+  has_reaction_baseline: 'False'
+  has_baseline: 'False'
+  cores: 6
+  detection_threshold: 0.1
```

Comment:
what is this number and why 0.1

Reply:
That is an extra parameter that is not being used by the algorithm, I can remove it from the configs

Reply:
Removed in 
aefc8f1